packages/next-auth/src/providers/shopify.js

```diff
@@ -0,0 +1,49 @@
+export default function Shopify(options) {
+  const { shop, scope, apiVersion = "2022-01" } = options
```

Comment:
As @balazsorban44 mentioned, `scope` [is no longer a top-level option](https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/providers/oauth.ts#L92-L148). Given this is a **JS** file you don't see any complaints, but in **TS** you wouldn't be allowed to read it like this ☕️

Instead, users of this provider can supply extra scopes like this:
```ts
  ShopifyProvider({
    clientId: process.env.SHOPIFY_ID,
    clientSecret: process.env.SHOPIFY_SECRET,
    authorization: { params: { scope: 'my-scopes' } }   
```
There's no need for you to handle this case in the code, as the options are going to be deeply merged on our side.